Tests/iaas/secure-connections/README.md

```diff
@@ -16,9 +16,11 @@ It is recommended to use a Python virtual environment[^2].
 Next, install the libraries required by the test suite:
 
 ```bash
-pip3 install -r requirements.txt
+pip3 install openstacksdk sslyze
 ```
 
+> Note: the version of the sslyze library determines the [version of the Mozilla TLS recommendation JSON](https://wiki.mozilla.org/Security/Server_Side_TLS#JSON_version_of_the_recommendations) that it checks against.
```

Comment:
Didn't we talk about somehow pinning this? My memory is admittedly fuzzy with regards to this discussion though.